model/license.go

```diff
@@ -265,6 +266,13 @@ func (l *License) IsPastGracePeriod() bool {
 	return timeDiff > LICENSE_GRACE_PERIOD
 }
 
+func (l *License) IsAroundSixtyDaysToExpiration() bool {
+	dif := l.ExpiresAt - GetMillis()
+	d, _ := time.ParseDuration(fmt.Sprint(dif) + "ms")
+	days := d.Hours() / 24
+	return days <= 60 && days >= 58
```

Comment:
Let's const these values, and you'll have to adjust the function name for IsAroundSixtyDaysToExpiration.